code/modules/mob/mob.dm

```diff
@@ -852,31 +852,40 @@ var/next_mob_id = 0
 /mob/proc/update_health_hud()
 	return
 
-/mob/living/on_varedit(modified_var)
-	switch(modified_var)
+/mob/living/vv_edit_var(var_name, var_value)
+	switch(var_name)
+		if("stat")
+			if((stat == 2) && (var_value < 2))//Bringing the dead back to life
+				dead_mob_list -= src
+				living_mob_list += src
+			if((stat < 2) && (var_value == 2))//Kill he
+				living_mob_list -= src
+				dead_mob_list += src
+	. = ..()
+	switch(var_name)
 		if("weakened")
-			SetWeakened(weakened)
+			SetWeakened(var_value)
```

Comment:
for atom colours you chose to use ```.color``` here, from the ```. = ..()``` part of the VV edit.
but here you choose to use ```var_value```, any particular reason? the inconsistency just seems odd.